app/packs/src/apps/mydb/elements/details/computeTasks/ComputeTaskContainer.js

```diff
@@ -67,30 +66,31 @@ export default class ComputeTaskContainer extends React.Component {
     const { tasks } = this.state;
 
     return (
-      <Panel className="eln-panel-detail" variant="primary">
-        <Panel.Heading>
-          {'Task'}
+      <Card className="eln-panel-detail">
```

Comment:
I think the CSS related to that class might be broken now. See cellLines.scss

Reply:
Adapted the class based on header and have kept the old classes intact until upgrade is finished. Will make a note in spreadsheet to clean up once all codes are refactored